configure.ac

```diff
@@ -74,9 +74,15 @@ AC_CHECK_HEADER(thenonexistentheader.h, HAVE_THENONEXISTENTHEADER_H="true")
 
 # set some ld -R nonsense
 #
+AC_ARG_ENABLE(rpath,
+[  --disable-rpath         do not hardcode runtime library paths],
+:, enable_rpath=yes)
```

Comment:
This looks backward to me: the option will be `--enable-rpath`, and if it is *not* given, `enable_rpath` is set to `yes`?

In any case, the variable `enable_rpath` should be set to `false` in the opposite case, to avoid parent processes' environment variables from bleeding through.